.github/workflows/stage-cf-deploy-lambda.yml

```diff
@@ -0,0 +1,32 @@
+### This is the Terraform-generated GitHub Actions workflow for cf-lambda    ###
+### applications for dev/stage/prod. It will publish the Lambda zip to S3    ###
+### and optionally re-apply Terraform in the Terraform Cloud                 ###
```

Comment:
I don't understand how this optionally re-applies TF... doesn't it always reapply? Or does this mean that if there were no changes TFC wouldn't apply because there were no changes?

Reply:
ahh... the "optional" is the `TF_AUTO_APPLY` flag -- if it's set to `false` then the workflow will run the `terraform apply` comment in TfC, but it won't approve the apply. But, in the end, this is another comment mistake -- I did not update the header to properly reflect what this specific workflow is doing.

Reply:
I think updating this comment is all that got missed. I'm still not clear what it means, but I think you agreed it wasn't accurate so I'll leave this unresolved until you either tell me it is correct as-is or push a change lol

Reply:
doh!